lib/screens/main_screen/bottom_bar.dart

```diff
@@ -71,23 +61,37 @@ class _BottomBarState extends State<BottomBar> {
     });
   }
 
-  @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
+  PreferredSizeWidget _appBarStyle() {
+    if (selectedIndex == 0) {
+      return AppBar(
+        backgroundColor: Colors.white,
+        title: Image.asset(
+          'assets/PICOS_Logo_RGB.png',
+          height: 70,
+        ),
+      );
```

Comment:
There are following problems with this solution:

- Padding and size of the Picos logo isn't correct.
- The logo shouldn't be on an AppBar in the first place. An AppBar always sticks to its position. But the mockup shows you clearly that you should be able to scroll away from the logo.